lib/customValidators.js

```diff
@@ -90,10 +91,11 @@ function validateChannelParams(parsedJSON, asyncapiYAMLorJSON, initialFormat) {
  * Validates if operationIds are duplicated in the document
  * 
  * @private
- * @param  {Object} parsedJSON parsed AsyncAPI document
- * @param  {String} asyncapiYAMLorJSON AsyncAPI document in string
- * @param  {String} initialFormat information of the document was oryginally JSON or YAML
- * @returns {Boolean} true in case the document is valid, otherwise throws ParserError
+ * @param {object} parsedJSON parsed AsyncAPI document
+ * @param {string} asyncapiYAMLorJSON AsyncAPI document in string
+ * @param {string} initialFormat information of the document was oryginally JSON or YAML
+ * @param operations
```

Comment:
this should be an array with names of all available channel operations, now it is `['publish', 'subscribe']`